### ISSUE TITLE
"appID required" error on user/group edit

### ISSUE THREAD

**Issue by idnaos** [woborschilde/fluentlogin#4]
When submitting user or group edit forms, no changes were made and the following error message appears:
```
Argument ''appID'' is required!
```
The system cannot gather field, group oder permission IDs, so the last URL shown in the address bar is:
```
https://intra.woborschil.net/fluentlogin/groupEdit.php?permission=on
```
As mentioned by the error message, the appID is also omitted without any known reason.

The error occurs on following hardware:
 - **Operating System:** Windows XP SP3
 - **Browser:** Mozilla Firefox 43.0.1

The error does **not** occur on Windows 10 (1607) with Firefox 53.

_This issue is archived and was originally created on 13 Aug 2017._

**Comment by idnaos** [comment]
**UPDATE:**

The error does **also** occur on Windows 10 (1607) with Firefox 53!<br />
Here it throws the following SweetAlert error message:
```
Fehler beim Aktualisieren der Benutzergruppen
<br /> <b>Notice</b>: Undefined variable: groupID in <b>/var/www/fluentlogin/functions/editGroup.php</b> on line <b>27</b><br />
<br /> <b>Notice</b>: Undefined variable: groupID in <b>/var/www/fluentlogin/functions/editGroup.php</b> on line <b>30</b><br /> 1
```

_15 Aug 2017_

**Comment by idnaos** [comment]
Fixed, will be included in next commit / Added in ee58de2c0ac9143964b0e8c70bf64db87dcfc4c4.

_Closed on 15 Aug 2017._
